### ISSUE TITLE
[4.0] Atum modules

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#33029]
The admin modules have various parameters in the advanced tab.

![image](https://user-images.githubusercontent.com/1296369/113627937-19445b80-965c-11eb-8f80-d6af1b809439.png)

However several of them have no impact at all as there is no code in the module chrome that will use them.

- bootstrap_size - no impact
- header_tag - no impact

Either the chrome needs to be updated to work or the params should be removed and then the following code from the chrome should also be removed

```
$bootstrapSize  = (int) $params->get('bootstrap_size', 6);
$moduleClass    = $bootstrapSize ? 'col-md-' . $bootstrapSize : 'col-md-12';
$headerTag      = htmlspecialchars($params->get('header_tag', 'h2'), ENT_QUOTES, 'UTF-8');

```

**Comment by Quy** [comment]
Related #25952

**Comment by brianteeman** [comment]
Just to add that the parameters do work if the chrome of "body" is selected instead of "well/default"

**Comment by brianteeman** [comment]
closed see #33045
